### PR TITLE
Fix documentation about wwi

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**

Some information about wwi wasn't up to date.

### CODE REVIEW

**Review by omichel — DISMISSED** [review]
Thank you.

File: docs/reference/robot.md [hunk 1]
```diff
@@ -2361,13 +2361,12 @@ The message is sent using the `webots.window("<robot window name>").send` method
...
+To read the full buffer, you should call repeatedly this function until it returns `NULL`:
 ```C
 const char *message;
 while ((message = wb_robot_wwi_receive_text())) {
   // do something with each message
 }
 ```
 The reading head will be reset to the beginning of the buffer of received messages each time a time step is performed.
-
-> **note** [Java, Python, MATLAB, ROS]: `wb_robot_wwi_receive` and `wb_robot_window_send` functions are not available in the Java, Python, MATLAB, or ROS API.
```
Comment (stefaniapedrazzi) [inline docs/reference/robot.md:2373]: Why did you remove this line?
It seems still correct that these functions are not available in Java Python MATLAB and ROS.

Comment (ad-daniel) [reply docs/reference/robot.md:2373]: Oh I see, it's the `receive` methods that are missing, all languages only have at least the `receive_text`, I must have read it wrongly then

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you!
